index.html

```diff
@@ -55,7 +55,8 @@ <h1 class="form-header">New Entry</h1>
         </div>
         <div class="row">
           <div class="form-button column-full">
-            <button id="btn" type="submit">Save</button>
+            <button id="delete-btn" class="link hidden">Delete Entry</button>
+            <button id="btn" class="btn-1" type="submit">Save</button>
```

Comment:
Because these buttons are in the same div, the delete button is the one that gets entered by default if editing the form. For example, if I am editing the title and press "enter" on my keyboard (as a shortcut to clicking the submit button), it defaults to popping up the delete entry modal.

One way to fix this would be to do what I have here and adding a class to `flex-direction: row-reverse` and place the submit button first so that is the first behavior when I use the "enter" key.
![Screenshot 2023-06-05 at 9 16 27 PM](https://github.com/vorleakyek/code-journal/assets/7886753/cbe7f860-234b-40f4-9227-ed8efedeeb02)

There are other ways of doing this but this could work.

Reply:
Thanks for pointing that out. That's a good case to check.